src/utility/Math.h

```diff
@@ -29,5 +29,14 @@ namespace Pinetime {
       }
       return res;
     }
+
+    constexpr int compileTimeAtoi(const char* str) {
```

Comment:
```suggestion
    constexpr int CompileTimeAtoi(const char* str) {
```
May as well fix the name while here :)